### DISCUSSION TITLE
Connection failures and no mouse or keyboard function

### DISCUSSION THREAD

**Discussion by tec81** [rustdesk/rustdesk#5927] [General]
### Bug Description

Since yesterday afternoon Rust Desk gets not connection to any computers I've tried. 
After installing the latest version 1.2.2 on Mac Intel and Apple Silicon I can only get connection if it's the same version.
If I have a connection there is a screen but no mouse functionality.
Checked all System Preferences and RustDesk settings. Have full access and keyboard and mouse active.

Tried different operating systems 12.7, 13.x and 14
Same problem when I try to use the iPhone app

### How to Reproduce

try to connect between Macs with RustDesk 1.2.2

### Expected Behavior

Connection between each Computer or device and Version of OS and RustDesk should be possible.

### Operating system(s) on local side and remote side

Mac OS 12.7

### RustDesk Version(s) on local side and remote side

1.2.2

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/4871#discussioncomment-6578012
